.github/workflows/claude.yml

```
@@ -1,27 +1,24 @@
 
 on:
-  issue_comment:
-    types: [created]
-  pull_request_review_comment:
-    types: [created]
-  issues:
-    types: [opened, assigned]
-  pull_request_review:
-    types: [submitted]
+  workflow_call:
+    #inputs: # 呼び出し側から渡されるパラメータ, 必要に応じて追加
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)